<launch>
  <group ns="urg1">
    <machine name="P2SEN-1" address="P2SEN-1" env-loader="/home/odroid/env.bash" user="odroid" password="odroid" timeout="30"/>

    <node machine="P2SEN-1" pkg="urg_node" name="urg_node" type="urg_node">
      <param name="ip_address" value="192.168.0.10"/>
      <param name="angle_min" value="-1.832595714595"/>
      <param name="angle_max" value=" 1.832595714595"/>
    </node>

    <node machine="P2SEN-1" pkg="tms_ss_pot" type="change_nan" name="change_nan">
      <remap from="/most_intense" to="/urg1/most_intense"/>
      <remap from="/LaserTracker" to="/LaserTracker1"/>
    </node>
  </group>

  <group ns="urg2">
    <machine name="P2SEN-2" address="P2SEN-2" env-loader="/home/odroid/env.bash" user="odroid" password="odroid" timeout="30"/>

    <node machine="P2SEN-2" pkg="urg_node" name="urg_node" type="urg_node">
      <param name="ip_address" value="192.168.0.10"/>
      <param name="angle_min" value="-1.832595714595"/>
      <param name="angle_max" value=" 1.832595714595"/>
    </node>

    <node machine="P2SEN-2" pkg="tms_ss_pot" type="change_nan" name="change_nan">
      <remap from="/most_intense" to="/urg2/most_intense"/>
      <remap from="/LaserTracker" to="/LaserTracker2"/>
    </node>
  </group>

  <group ns="urg3">

    <machine name="P2SEN-3" address="P2SEN-3" env-loader="/home/odroid/env.bash" user="odroid" password="odroid" timeout="30"/>

    <node machine="P2SEN-3" pkg="urg_node" name="urg_node" type="urg_node">
      <param name="ip_address" value="192.168.0.10"/>
      <param name="angle_min" value="-1.832595714595"/>
      <param name="angle_max" value=" 1.832595714595"/>
    </node>

    <node machine="P2SEN-3" pkg="tms_ss_pot" type="change_nan" name="change_nan">
      <remap from="/most_intense" to="/urg3/most_intense"/>
      <remap from="/LaserTracker" to="/LaserTracker3"/>
    </node>

  </group>

  <group ns="urg4">

    <machine name="P2SEN-4" address="P2SEN-4" env-loader="/home/odroid/env.bash" user="odroid" password="odroid" timeout="30"/>

    <node machine="P2SEN-4" pkg="urg_node" name="urg_node" type="urg_node">
      <param name="ip_address" value="192.168.0.10"/>
      <param name="angle_min" value="-1.832595714595"/>
      <param name="angle_max" value=" 1.832595714595"/>
    </node>

    <node machine="P2SEN-4" pkg="tms_ss_pot" type="change_nan" name="change_nan">
      <remap from="/most_intense" to="/urg4/most_intense"/>
      <remap from="/LaserTracker" to="/LaserTracker4"/>
    </node>

  </group>    
</launch>
